patches/src/main/kotlin/app/revanced/patches/angulus/RemoveAdsPatch.kt

```diff
@@ -0,0 +1,18 @@
+package app.revanced.patches.angulus
+
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.returnEarly
+
+@Suppress("unused")
+val angulusPatch = bytecodePatch(
+    name = "Hide ads",
+    description = "Hides in app ads",
+) {
+    compatibleWith("com.drinkplusplus.angulus"("5.0.20"))
+
+    execute {
+        // Always returns 0 as the daily measurement count
```

Comment:
What is a "measurement"?

```suggestion
        // Always return 0 as the daily measurement count.
```